README.adoc

```diff
@@ -171,6 +178,15 @@ The metadata is inserted into the log for the device, and also as a serial-numbe
 
 Set to any value to enable metadata collection.
 
+=== RPI_DEVICE_RETRIEVE_PRIVATE_KEY
+*Optional*
```

Comment:
mention this is not possible if CUSTOMER_KEY_PKCS11_NAME  is used?

Reply:
> oh. is the device unique key the symmetric key used for the LUKSv2 encryption? Having _PRIVATE_ in the name makes it sound like an asymmetric key.

The device unique key is indeed asymmetric - but only the private component is retained by the device. This component is used as the unlock passphrase for LUKS - but can also be used for other purposes by trusted software, where having the public key stored may allow you to authenticate a given device.